.github/workflows/i18n-crowdin-download.yml

```diff
@@ -113,3 +113,9 @@ jobs:
 
         env:
           GITHUB_TOKEN: ${{ steps.generate_token.outputs.token }}
+
+      - name: Run auto-milestone
+        uses: grafana/grafana-github-actions-go/auto-milestone@main
+        with:
```

Comment:
You need a `if: steps.crowdin-download.outputs.pull_request_url` check on here also to only run this if a PR was created.